.github/workflows/cypress.yml

```diff
@@ -118,6 +119,47 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: 'us-gov-west-1'
 
+      - name: Get Slack app token
+        if: ${{ needs.build.result == 'failure' }}
+        uses: marvinpinto/action-inject-ssm-secrets@v1.1.1
+        with:
+          ssm_parameter: /devops/github_actions_slack_socket_token
+          env_variable_name: SLACK_APP_TOKEN
+
+      - name: Get Slack bot token
+        if: ${{ needs.build.result == 'failure' }}
+        uses: marvinpinto/action-inject-ssm-secrets@v1.1.1
+        with:
+          ssm_parameter: /devops/github_actions_slack_bot_user_token
+          env_variable_name: SLACK_BOT_TOKEN
+
+      - name: Get va-vsp-bot token
+        uses: marvinpinto/action-inject-ssm-secrets@v1.1.1
+        with:
+          ssm_parameter: /devops/VA_VSP_BOT_GITHUB_TOKEN
+          env_variable_name: VA_VSP_BOT_GITHUB_TOKEN
+
+      - name: Checkout VSP actions
+        if: ${{ needs.build.result == 'failure' }}
+        uses: actions/checkout@v2
+        with:
+          repository: department-of-veterans-affairs/vsp-github-actions
+          ref: refs/heads/main
+          token: ${{ env.VA_VSP_BOT_GITHUB_TOKEN }}
+          persist-credentials: false
+          path: ./.github/actions/vsp-github-actions
+
+      - name: Notify Slack
+        if: ${{ needs.build.result == 'failure' }}
+        uses: ./.github/actions/vsp-github-actions/slack-socket
+        env:
+          SSL_CERT_DIR: /etc/ssl/certs
+        with:
+          slack_app_token: ${{ env.SLACK_APP_TOKEN }}
+          slack_bot_token: ${{ env.SLACK_BOT_TOKEN }}
+          attachments: '[{"mrkdwn_in": ["text"], "color": "danger", "text": "E2E tests in `vets-website` have failed, run: <https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}|${{github.run_id}}>"}]'
+          channel_id: ${{ env.CHANNEL_ID }}
```

Comment:
Would it make more sense for the Slack notification to happen in the same job as the test? This doesn't really seem related to the Mochawesome report.

Reply:
I placed it in the Mochawesome report job due to it only needing to send one notification per deployment.  Since the first job runs in a matrix, it would send 6 duplicate notifications that are all identical which I felt was probably not the best approach.  I could create a separate job for it if we want to separate it for proper categorization, or maybe you know a way to make it only run once per matrix? I was unable to find a way to do this.

Reply:
Ah, okay. That makes sense; for some reason I was thinking it would only handle one of the failures, but forgot `fail-fast` is disabled for those tests.